src/components/color-picker/color-picker.e2e.ts

```diff
@@ -319,7 +319,7 @@ describe("calcite-color-picker", () => {
   };
 
   function assertUnsupportedValueMessage(value: string | object | null, format: string): void {
-    expect(consoleSpy).toBeCalledTimes(1);
+    expect(consoleSpy).toHaveBeenCalledTimes(1);
```

Comment:
This is unrelated, please revert.

Reply:
Its' the pre-commit linting job doing the change. Once #5431 lands-in, it will disappear.

Reply:
Oh, I see. Ignore my comment then.